server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```
@@ -355,6 +355,9 @@ public void doRun()
               }
               commitRunnable.run();
             }
+            catch (Exception e) {
+              metrics.incrementFailedPersists();
```

Did you mean to suppress the exception here?